tests/publishers/conftest.py

```diff
@@ -2,15 +2,36 @@
 from uuid import UUID
 
 import pytest
+import responses
 
 from mobilizon_reshare.event.event import MobilizonEvent
 from mobilizon_reshare.publishers.abstract import AbstractPublisher
 from mobilizon_reshare.publishers.exceptions import PublisherError, InvalidResponse
 
+api_uri = "https://zulip.twc-italia.org/api/v1/"
```

Comment:
all this stuff is specific to zulip so it shouldn't be in the conftest for all the publishers, especially if it has generic names. Move it to test_zulip or create a folder for all the zulip stuff